src/shortcuts/plugin.ts

```diff
@@ -245,6 +245,16 @@ const SHORTCUTS = [
     selector: '.jp-Console .jp-CellEditor',
     sequence: ['Escape']
   },
+  {
+    command: 'terminal:increase-font',
+    selector: '.jp-TerminalWidget .jp-TerminalWidget-body',
+    sequence: ['Ctrl =']
```

Comment:
-1 on default overriding the standard browser shortcuts for increasing/decreasing sizes. But if we're going to do it, we should at least make terrible on all platforms :).